pkg/env/costmodelenv.go

```diff
@@ -0,0 +1,163 @@
+// the AWS access key secret for authentication
+func GetAWSAccessKeySecret() string {
+	return Get(AWSAccessKeySecretEnvVar, "")
+}
+
+// GetAWSClusterID returns the environment variable value for AWSClusterIDEnvVar which represents
+// an AWS specific cluster identifier.
+func GetAWSClusterID() string {
+	return Get(AWSClusterIDEnvVar, "")
+}
+
+// GetKubecostNamespace returns the environment variable value for KubecostNamespaceEnvVar which
+// represents the namespace the cost model exists in.
+func GetKubecostNamespace() string {
+	return Get(KubecostNamespaceEnvVar, "kubecost")
+}
+
+// GetClusterID returns the environment variable value for ClusterIDEnvVar which represents the
+// configurable identifier used for multi-cluster metric emission.
+func GetClusterID() string {
+	return Get(ClusterIDEnvVar, "")
+}
+
+// GetPrometheusServerEndpoint returns the environment variable value for PrometheusServerEndpointEnvVar which
+// represents the prometheus server endpoint used to execute prometheus queries.
+func GetPrometheusServerEndpoint() string {
+	return Get(PrometheusServerEndpointEnvVar, "")
+}
+
+// IsRemoteEnabled returns the environment variable value for RemoteEnabledEnvVar which represents whether
+// or not remote write is enabled for prometheus for use with SQL backed persistent storage.
+func IsRemoteEnabled() bool {
+	return GetBool(RemoteEnabledEnvVar, false)
+}
+
+// GetRemotePW returns the environment variable value for RemotePWEnvVar which represents the remote
+// persistent storage password.
+func GetRemotePW() string {
+	return Get(RemotePWEnvVar, "")
+}
+
+// GetSQLAddress returns the environment variable value for SQLAddressEnvVar which represents the SQL
+// database address used with remote persistent storage.
+func GetSQLAddress() string {
+	return Get(SQLAddressEnvVar, "")
+}
+
+// IsUseCSVProvider returns the environment variable value for UseCSVProviderEnvVar which represents
+// whether or not the use of a CSV cost provider is enabled.
+func IsUseCSVProvider() bool {
+	return GetBool(UseCSVProviderEnvVar, false)
+}
+
+// GetCSVRegion returns the environment variable value for CSVRegionEnvVar which represents the
+// region configured for a CSV provider.
+func GetCSVRegion() string {
+	return Get(CSVRegionEnvVar, "")
+}
+
+// GetCSVPath returns the environment variable value for CSVPathEnvVar which represents the key path
+// configured for a CSV provider.
+func GetCSVPath() string {
+	return Get(CSVPathEnvVar, "")
+}
+
+// GetConfigPath returns the environment variable value for ConfigPathEnvVar which represents the cost
+// model configuration path
+func GetConfigPath() string {
+	return Get(ConfigPathEnvVar, "")
+}
+
+// GetConfigPath returns the environment variable value for ConfigPathEnvVar which represents the cost
+// model configuration path
+func GetConfigPathWithDefault(defaultValue string) string {
```

Comment:
Do you think we could get away with one function called `GetConfigPath()` that defaulted to some constant in the package whose value is `"/models/"`? The reasoning being that the only places where `GetConfigPath()` is currently used are commented-out, and everywhere this function is used, the same default is given.

(I don't feel strongly about this, just playing devil's advocate.)

Reply:
You're probably right, but was just trying to set a convention for any env vars in the future that would need a reasonable default passed in.